b2/account_info/abstract.py

```diff
@@ -137,3 +182,20 @@ def take_large_file_upload_url(self, file_id):
     @abstractmethod
     def clear_large_file_upload_urls(self, file_id):
         pass
+
+    @staticmethod
+    def allowed_is_valid(allowed):
+        """
+        Returns true iff the 'allowed' dict passed in is legal.
```

Comment:
iff

Reply:
"iff" == "if and only if"

I'll spell it out.